api/src/services/taxonomy-service.ts

```diff
@@ -56,13 +56,12 @@ export class TaxonomyService {
   }
 
   _sanitizeTaxonRecordsData(taxonRecords: TaxonRecord[]): TaxonSearchResult[] {
-    return taxonRecords.map((item: TaxonRecord) => {
-      return {
-        tsn: item.itis_tsn,
-        commonName: item.common_name,
-        scientificName: item.itis_scientific_name
-      };
-    });
+    return taxonRecords.map((item: TaxonRecord) => ({
+      tsn: item.itis_tsn,
+      // placeholder: wrap commonNames in array until the database supports multiple common names
```

Comment:
This comment is useful but I would mark it with `TODO` so it can be easily found during future tech debt efforts.

To that end, if there is an existing Jira ticket that tracks this desired change (namely, **add database support for multiple common names**), it's a good idea to mention the ticket number or provide a link to the Jira ticket.